Source/ENS/Private/Characters/Player/EnsPlayerController.cpp

```diff
@@ -154,3 +238,39 @@ void AEnsPlayerController::Interact()
 			Cast<AEnsPlayerCharacter>(GetCharacter())->MoveToActor(Hit.GetActor());
 	}
 }
+
+bool AEnsPlayerController::GetChildsOfActor(const AActor* Actor, TArray<UStaticMeshComponent*>& Result)
+{
+	if (!Actor)
+		return false;
+	
+	AActor* Parent = Actor->GetAttachParentActor();
+	if (Parent)
```

Comment:
invert the if and return tmp->FindComponentByClass<UStaticMeshComponent>() since result will be empty, or add by range instead of a for loop.